### ISSUE TITLE
Don't use expect_setequal() in linter tags test

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#1018]
```
lintr_db$linter[10] absent from `all_linters`
── Failure (test-linter_tags.R:26:3): available_linters matches the set of linters available from lintr ──
`all_linters`[11] absent from lintr_db$linter
```

The failure note is pretty unhelpful

**Comment by AshesITR** [comment]
`expect_equal()` with the `platform_independent_sort()` of all exported symbols should give a nice error message.
